src/store/mod.rs

```diff
@@ -121,6 +125,12 @@ pub struct StoredProp<S = String, V = AstarteType> {
     pub ownership: Ownership,
 }
 
+/// A property that may be unset.
+///
+/// This is returned by getting all the properties (`load_all_props`) that have not been deleted
+/// yet, since they where not sent to Astarte.
+pub type MaybeStoredProp = StoredProp<String, Option<AstarteType>>;
```

Comment:
I would like to suggest a different name for `MaybeStoredProp`:
- `NullableStoredProp`
- `OptionalStoredProp`
- ...

Reply:
Since it represents a property that could be  `Set or Unset`, I think a name with `Opt` is better.
```suggestion
pub type OptStoredProp = StoredProp<String, Option<AstarteType>>;
```

Reply:
I vote for `OptStoredProp` I think it represents better than `PropWithUnset`.